lib/model-interfaces/langchain/functions/request-handler/adapters/bedrock/base.py

```diff
@@ -107,47 +138,103 @@ class BedrockChatNoStreamingAdapter(BedrockChatAdapter):
     """Some models do not support system streaming using the converse API"""
 
     def __init__(self, *args, **kwargs):
+        logger.info(
+            "Initializing BedrockChatNoStreamingAdapter with disabled streaming."
+        )
         super().__init__(disable_streaming=True, *args, **kwargs)
 
 
 class BedrockChatNoSystemPromptAdapter(BedrockChatAdapter):
-    """Some models do not support system and message history in the conversion API"""
+    """Some models do not support system and message history in the conversation API"""
 
     def get_prompt(self):
-        template = """The following is a friendly conversation between a human and an AI. If the AI does not know the answer to a question, it truthfully says it does not know.
+        # Fetch the conversation prompt and translated
+        # words based on the current language
+        conversation_prompt = prompts[locale]["conversation_prompt"]
+        question_word = prompts[locale]["question_word"]
+        assistant_word = prompts[locale]["assistant_word"]
+        logger.info("Generating no-system-prompt template for conversation.")
+
+        # Combine conversation prompt, chat history, and input into the template
+        template = f"""{conversation_prompt}
+
+{{chat_history}}
 
-Current conversation:
-{chat_history}
+{question_word}: {{input}}
 
-Question: {input}
+{assistant_word}:"""
 
-Assistant:"""  # noqa: E501
-        return PromptTemplateWithHistory(
-            template=template, input_variables=["input", "chat_history"]
+        # Create the PromptTemplateWithHistory instance
+        prompt_template = PromptTemplateWithHistory(
+            input_variables=["input", "chat_history"], template=template
         )
 
+        # Log the content of PromptTemplateWithHistory before returning
+        logger.debug(f"PromptTemplateWithHistory template: {prompt_template.template}")
+
+        return prompt_template
+
     def get_condense_question_prompt(self):
-        template = """Given the following conversation and a follow up question, rephrase the follow up question to be a standalone question, in its original language.
-
-Chat History:
-{chat_history}
-Follow Up Input: {input}
-Standalone question:"""  # noqa: E501
-        return PromptTemplateWithHistory(
-            template=template, input_variables=["input", "chat_history"]
+        # Change le niveau global à DEBUG
+        # Fetch the prompt and translated words based on the current language
+        condense_question_prompt = prompts[locale]["condense_question_prompt"]
+        logger.info(f"condense_question_prompt: {condense_question_prompt}")
```

Comment:
```suggestion
        logger.debug(f"condense_question_prompt: {condense_question_prompt}")
```
I would recommend to mark them all as debug to reduce cloudwatch usage. (nitpick sorry)